tests/util/test_helpers.py

```diff
@@ -0,0 +1,305 @@
+"""Test data as functions and common assertion helper functions."""
+from typing import Dict, Union
+
+from shipengine_sdk import ShipEngine
+from shipengine_sdk.models import Address, AddressValidateResult, Endpoints
+
+
+def stub_config() -> Dict[str, any]:
+    """
+    Return a test configuration dictionary to be used
+    when instantiating the ShipEngine object.
+    """
+    return dict(
+        api_key="baz", base_uri=Endpoints.TEST_RPC_URL.value, page_size=50, retries=2, timeout=15
+    )
+
+
+def stub_shipengine_instance() -> ShipEngine:
+    """Return a test instance of the ShipEngine object."""
+    return ShipEngine(stub_config())
+
+
+def valid_residential_address() -> Address:
+    """
+    Return a test Address object with valid residential
+    address information.
+    """
+    return Address(
+        street=["4 Jersey St", "Apt. 2b"],
+        city_locality="Boston",
+        state_province="MA",
+        postal_code="02215",
```

Comment:
@KaseyCantu if I understand this ticket correctly, shouldnt this fixture be generating a numeric postal code? https://auctane.atlassian.net/browse/DX-1045?atlOrigin=eyJpIjoiNDk2M2Y0MjJjNWUzNDVhNjgyOGEyYWFmNDEyMTA1YzYiLCJwIjoiaiJ9

Reply:
@pierce-h  Are you asking if this postal code should be an integer instead of a string?

Reply:
I have confirmation from James that this field should always be a string, and in the tests for numeric postal code and alpha postal code, we are checking that the chars in the string of the postal code are the correct types of chars. This is why I am using a regex to validate these strings are numeric/alpha-numeric in the Models.